api/app/routers/playthroughs.py

```
@@ -0,0 +1,266 @@
+    ValidationError,
+)
+from app.services.playthroughs_service import PlaythroughsService
+
+router = APIRouter(prefix="/playthroughs", tags=["playthroughs"])
+
+
+@router.get("", response_model=PlaythroughListResponse)
+async def list_playthroughs(
+    status: Optional[list[PlaythroughStatus]] = Query(
+        None, description="Filter by playthrough status"
+    ),
```

Ruff (0.12.2)

Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)